cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketDelegator.java

```diff
@@ -73,7 +75,7 @@ public final TicketGrantingTicket getGrantingTicket() {
     }
 
     @Override
-    public final long getCreationTime() {
+    public final ZonedDateTime getCreationTime() {
```

Comment:
I added a util class to handle the transfer to something JPA understands. Which is what hibernate will be doing to since JPA wasn't updated to the new APIs.